### ISSUE TITLE
Version - User polymorphic relationship

### ISSUE THREAD

**Issue by MartinPetricko** [overtrue/laravel-versionable#73]
First of all thanks for creating this package. Love how simple and useful it is! ❤️

I would like to propose a change in how version author is stored. At the moment there is no simple way to keep versions from multiple user models. Some applications can have multiple guards with different model providers, while sharing some of the resources. For example we can have `Admin`, `User` and both of them can edit an `Article`.

Would there be any problem with changing Version - User relationship from `belongsTo` to `morphTo`?

**Comment by overtrue** [comment]
Yes, indeed, the current version doesn't work for your scenario, but I can look into the next major version 6.x to support your scenario, and for now, you can consider modifying migrations + inheriting the default Version class to do so.

Thanks.
